KuringLite/Search/View/SearchView.swift

```diff
@@ -35,36 +35,46 @@ import KuringCommons
 
 /// 검색화면을 나타내는 뷰
 struct SearchView: View {
-    @StateObject private var engine = SearchEngine()
+    @EnvironmentObject var engine: SearchEngine
```

Comment:
`@EnvironmentObject` 가 바인딩하고자 하는 데이터모델 인스턴스를 찾을 수가 없습니다.
```suggestion
    @StateObject private var engine = SearchEngine()
```
`@StateObject` 속성을 사용하여 데이터모델 인스턴스를 생성합니다.

Reply:
개선했습니다.